srcpkgs/subuser/template

```diff
@@ -1,11 +1,11 @@
 # Template file for 'subuser'
 pkgname=subuser
 version=0.6.2
-revision=1
+revision=2
 only_for_archs="x86_64 x86_64-musl"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="docker python3"
+depends="docker python3 python3-setuptools"
```

Comment:
You can omit `python3` then. It's automatically installed with `python3-setuptools`

Reply:
Also, change the license to `LGPL-3.0-only`